spec/restaurant_spec.rb

```diff
@@ -0,0 +1,12 @@
+require 'spec_helper'
+require 'restaurant'
+describe 'Restaurant' do
+
+   subject(:restaurant) { described_class.new }
+
+
+   it "there is a menu" do
+      expect(Restaurant::MENU).to_not be_empty
+    end
```

Comment:
Lint/BlockAlignment: `end` at 10, 4 is not aligned with `it "there is a menu" do` at 8, 3.